docs/sources/features/datasources/loki.md

```diff
@@ -92,6 +92,20 @@ Example queries:
 * `{name="kafka"} tsdb-ops.*io:2003`
 * `{instance=~"kafka-[23]",name="kafka"} kafka.server:type=ReplicaManager`
 
+## Live Tailing
+
+To view your logs live as they are added, choose `Live` from the refresh dropdown, and you should see your logs be displayed in real time.
```

Comment:
I'd add:

Live Tailing relies on two Websocket connections: between the browser and the Grafana server, as well as between the Grafana server and the Loki server. If you run any reverse proxies, please configure them accordingly.